### DISCUSSION TITLE
What is the sorting of documents of an account (organization)?

### DISCUSSION THREAD

**Discussion by manuxi** [sulu/sulu#7301] [General]
I have stored 3 logos in an account: the one you see on the details page and two more under documents. It's about these last two, more precisely about the sorting. 

One of them has a white font (called A), one has a dark font (called B). A was added first, B second.

(Both are needed because I have a light mailbody but a dark footer background where they should be displayed.)

My code:
```
{% set organisation = sulu_snippet_load_by_area('organisation') %}
{% set account = organisation.content.account.entity %}
{% set additionalImages = account.getMedias() %}
```

Now, strangely enough, additionalImages[0] is image B, additionalImages[1] is image A.

What is the sorting based on? Can I rely on the current sorting?

**Comment by alexander-schranz** [comment]
The sorting is currently based on the `id` https://github.com/sulu/sulu/blob/a57671127bde0227699c4e609052a514ecba9c98/src/Sulu/Bundle/ContactBundle/Resources/config/doctrine/Account.orm.xml#L104 which maybe was a bad idea as now all queries automatically fetching accounts are adding `order by account_media.id ASC` to it which even could make quering a list of accounts slow. So maybe will change in the future.
